monkey/common/agent_events/agent_shutdown_event.py

```diff
@@ -0,0 +1,12 @@
+from . import AbstractAgentEvent
+
+
+class AgentShutdownEvent(AbstractAgentEvent):
+    """
+    An event that occurs when the Agent shuts down
+
+    Attributes:
+        :param stop_time: The time that the Agent was shut down (seconds since the Unix epoch)
+    """
+
+    stop_time: float
```

Comment:
Since `AbstractAgentEvent` already has a timestamp, do we need a separate stop time? Maybe this whole class can be reduced to

```

class AgentShutdownEvent(AbstractAgentEvent):
    """
    An event that occurs when the Agent shuts down
    """
```